worker/agentconfigupdater/worker.go

```diff
@@ -108,10 +106,7 @@ func (w *agentConfigUpdater) onConfigChanged(topic string, data controllermsg.Co
 	nonSyncedWritesToRaftLog := data.Config.NonSyncedWritesToRaftLog()
 	nonSyncedWritesToRaftLogChanged := nonSyncedWritesToRaftLog != w.nonSyncedWritesToRaftLog
 
-	batchRaftFSM := data.Config.BatchRaftFSM()
-	batchRaftFSMChanged := batchRaftFSM != w.batchRaftFSM
-
-	changeDetected := mongoProfileChanged || jujuDBSnapChannelChanged || nonSyncedWritesToRaftLogChanged || batchRaftFSMChanged
+	changeDetected := mongoProfileChanged || jujuDBSnapChannelChanged || nonSyncedWritesToRaftLogChanged
```

Comment:
That last also needs to be removed from config.

Reply:
I'll make a follow up PR after this lands.